README.md

```diff
@@ -173,7 +173,8 @@ Randomizes the capitalization of returned domain names.
 ### Custom OpenSSL
 
 Disabled by default.
-Enabled with `--prefix-openssl=/path/to/openssl`.
+Enabled with `--prefix-openssl=/path/to/openssl` or
+`--openssl-inc=/path/to/openssl_inc` or `--openssl-lib=/path/to/openssl_lib`
```

Comment:
Should it be "and" instead of "or" on line 177 (updated text) to match the statement on line 190-191 (updated text):
```
same parent directory, use `--openssl-inc` and `--openssl-lib` options to
specify both paths.
```

Reply:
I didn't want to restrict the usage of the options. One could use them separately. Hence the "or" line 177. About the documentation line 190-191, I think the meaning is also correct. You need both options to "specify both paths", hence the "and". Are you saying that this is confusing?

Reply:
I think the text in 189-191 says that both should be set if not headers and libraries are in the same directory.

Reply:
Which is how I expect people to use such options. But I again I didn't want to restrict the usage. The line 177 is the documentation of the following line in Makefile.PL:

https://github.com/zonemaster/zonemaster-ldns/blob/f39abb9ccc2676254a3e86251fa5b207d8a6d19d/Makefile.PL#L97

Reply:
I think you say "and" above but "or" in the documentation.

You have added a lot of POD docuementation into Makefile.PL, which we do not have in other repositories. The documentation overlaps with the documentation in the README.md file. Wouldn't it be better to put the documentation in one place and then refer from the other?

Reply:
> which we do not have in other repositories

Looking at other repositories, I don't think we provide arguments to pass options to Makefile.PL.

> Wouldn't it be better to put the documentation in one place and then refer from the other?

I guess this would be better indeed. I'd say it would be better to have it close to the code so that you can refer to it easily without needing several files. And you can run `perldoc Makefile.PL` to access the documentation.

Reply:
I think it would be fine to state "run `perldoc Makefile.PL`" in README.md.

Reply:
If there are no conflicts between them there is at least no short-term harm. So that is OK.